src/tests/sbus_codegen_tests.c

```diff
@@ -115,15 +115,15 @@ END_TEST
 
 START_TEST(test_signals)
 {
-    const struct sbus_signal_meta *signal;
+    const struct sbus_signal_meta *sm_signal;
```

Comment:
I would prefer `ssm_singal` (Sbus Signal Meta) or `sgnl` `sig`

Reply:
And you kind of agreed in https://github.com/SSSD/sssd/pull/50#discussion_r87221861